apps/gitness/src/components/FileExplorer.tsx

```diff
@@ -88,7 +87,7 @@ export default function Explorer({ selectedBranch, repoDetails }: ExplorerProps)
     const sortedEntries = sortEntriesByType(entries)
     return sortedEntries.map((item, idx) => {
       const itemPath = parentPath ? `${parentPath}/${item.name}` : item.name
-      const fullPath = `/${spaceId}/repos/${repoId}/code/${selectedBranch}/~/${itemPath}`
+      const fullPath = `/${spaceId}/repos/${repoId}/code/${fullGitRef ?? selectedBranch}/~/${itemPath}`
```

Comment:
Is there any chance we could do a relative path here ?

Reply:
sure